src/chapters/chapter1.md

```diff
@@ -1,19 +1,39 @@
+She'd refused to let herself relax since she first called the police to report that their father was missing. In her brother's arms, however, her body no longer seemed to listen and she let out a small sigh, "It's so good to see you J.J. Thank you for coming."
 
-John parked in the driveway, grabbed his overnight bag from the backseat - he'd assured Alexa that the trip would only be a day or two and when he got back they could talk through everything - and got out of the car.
+"Of course. Let's get out of the cold and you can catch me up on what's going on."
 
-Amy must have heard the tires spinning on the loose gravel because when John looked up she was standing in the doorway. Under different circumstances she would have smiled. John failed to suppress his own smile which crept into the corners of his lips and inched them upwards as he neared the door. Hers was the first friendly face he'd seen in what seemed like weeks. He pulled the door open, dropped his duffel on the threshold and enveloped her slight frame in his massive arms, "How you doing, kid?"
+# Chapter 2
 
-Her body responded reflexively by sagging. Safe and secure in her big brother's arms, it rebelled by relaxing, something she'd not allowed herself since she'd first called the police to report their father was missing.
+It took a moment for his eyes to adjust to the light - or lack there of. No one had bothered to turn any lights on in the living room, so the only light was the little that managed to sneak past the tightly drawn curtains. His hand instinctively reached for the switch on the wall, but no lights came on.
 
-# Chapter 2
+"Burnt out."
+
+"Huh, strange. I know dad's always been mostly useless around the house, but to let all of the lights burn out and not change them feels... well, it feels unusual. The dust on the other hand - that's no surprise." Every horizontal surface was caked with dust. That's how it'd been ever since mom had passed - or would have been had it not been for Patrick, Amy, and him. Patrick had moved to New York years ago, and he hadn't stepped foot in the house for even longer, so taking care of dad had fallen on Amy - not that she didn't have plenty on her plate as it is.
+
+"Yeah, he's been even more _himself_ of late. Working all the time. It's been all I can do to get him to come over for Sunday dinner with me and Beau." Dad had always been a workaholic. Growing up he'd stay at the lab until well after the family had sat down for dinner. When he did finally get home, he'd head straight to his study, where he'd pour himself a whisky and light a cigar to bury himself in more work, but only after closing the big oak door.
```

Comment:
Does the insertion of the exposition/character development in this paragraph feel a little bit forced or out of place somehow? I like the specificity of the details.